src/components/ChannelList/ChannelList.js

```diff
@@ -352,6 +356,12 @@ ChannelList.propTypes = {
    * @param {Event}     event       [Event object](https://getstream.io/chat/docs/event_object/?language=js) corresponding to `channel.deleted` event
    * */
   onChannelDeleted: PropTypes.func,
+  /**
+   * Optional function to filter channels prior to loading in the DOM. Do not use any complex or async logic here that would significantly delay the loading of the ChannelList. We recommend array methods like filter/sort/reduce.
```

Comment:
you can prob newline this comment so it's a bit easier for us to read

Reply:
also mention this function should be a pure function without any data mutations